worlds/animal_well/bean_patcher.py

```diff
@@ -1104,6 +1120,47 @@ def apply_deathlink_patch(self):
         if bean_died_trampoline.apply():
             self.revertable_patches.append(bean_died_trampoline)
 
+    def write_protected_memory(self, addr, data):
+        page = ctypes.c_ulonglong(addr & ~0xFFF)
+        old_protect = pymem.memory.virtual_query(self.process.process_handle, addr).Protect
+        pymem.ressources.kernel32.VirtualProtectEx(self.process.process_handle, page, 0x1000, 0x40)
+        self.process.write_bytes(addr, data, len(data))
+        pymem.ressources.kernel32.VirtualProtectEx(self.process.process_handle, page, 0x1000, old_protect)
+
+    def write_save_file_name(self, seeded_save_file="AnimalWell.sav"):
+        addr = self.module_base + 0x20ac5e0
+        self.write_protected_memory(addr, seeded_save_file.encode("utf-16le") + b"\x00\x00")
+        self.process.write_bytes(self.application_state_address + 0x400 + 0x750cc, b"\x01", 1) # return to title screen to reload new save file
```

Comment:
```suggestion
        self.write_protected_memory(addr, seeded_save_file.encode("utf-16le") + b"\x00\x00")
        # return to title screen to reload new save file
        self.process.write_bytes(self.application_state_address + 0x400 + 0x750cc, b"\x01", 1)
```
Either this or a double space between the end of the line and the # at the start of the comment.
Same for the other ones where the comment is in-line